marshmallow/fields.py

```diff
@@ -938,7 +938,6 @@ def _serialize(self, value, attr, obj):
             return value.isoformat()
         except AttributeError:
             self.fail('format', input=value)
-        return value
```

Comment:
Why was this removed? It seems unrelated to the feature.

This was returning `value` if an exception other than `AttributeError` occurred.